software/deploy/deploy.go

```diff
@@ -289,3 +293,86 @@ func getAirflowUILink(deploymentID string, deploymentURLs []houston.DeploymentUR
 	}
 	return ""
 }
+
+func isDagOnlyDeploymentEnabled(appConfig *houston.AppConfig) bool {
+	return appConfig != nil && appConfig.Flags.DagOnlyDeployment
+}
+
+func isDagOnlyDeploymentEnabledForDeployment(deploymentInfo *houston.Deployment) bool {
+	return deploymentInfo != nil && deploymentInfo.DagDeployment.Type == houston.DagOnlyDeploymentType
+}
+
+func validateIfDagDeployURLCanBeConstructed(deploymentInfo *houston.Deployment) error {
+	_, err := config.GetCurrentContext()
+	if err != nil {
+		return fmt.Errorf("could not get current context! Error: %w", err)
+	}
+	if deploymentInfo == nil || deploymentInfo.ReleaseName == "" {
+		return errInvalidDeploymentID
+	}
+	return nil
+}
+
+func getDagDeployURL(deploymentInfo *houston.Deployment) string {
+	c, _ := config.GetCurrentContext()
+	return fmt.Sprintf("deployments.%s/%s/upload", c.Domain, deploymentInfo.ReleaseName)
+}
+
+func DagsOnlyDeploy(houstonClient houston.ClientInterface, appConfig *houston.AppConfig, deploymentID, dagsParentPath string, dagDeployURL *string, cleanUpFiles bool) error {
```

Comment:
is there a way for user to pass this url? - dagDeployURL

i dont see any flag that allows for this

Reply:
This URL is auto-generated. Customers don't have the option to specify this as of now.